public/sass/_variables.dark.scss

```diff
@@ -3,25 +3,10 @@
 
 $theme-name: dark;
 
-// New Colors
-// -------------------------
-$sapphire-faint: #041126;
-$sapphire-light: #5794F2;
-$sapphire-base: #3274D9;
-$sapphire-shade: #1F60C4;
-$lobster-base: #E02F44;
-$lobster-shade: #C4162A;
-$forest-light: #96D98D;
-$forest-base: #37872D;
-$forest-shade: #19730E;
-$green-base: #299C46;
-$green-shade: #23843B;
-
-
 // Grays
 // -------------------------
-$black: #000;
-$dark-1: #141414;
+$black: #000000;
+$dark-1: #1f1f20;
```

Comment:
$dark-1 and $dark-2 is the same color?

Reply:
@jschill variables need sync with master before merging as @Ijin08 is working on #15411